src/IntercomMessages.php

```diff
@@ -19,4 +19,45 @@ public function create($options)
     {
         return $this->client->post("messages", $options);
     }
+
```

Comment:
The message export feature is on the `Unstable` version at the moment and is subject to changes, so I am not sure if we want to add something not stable to the SDK 🤔 @mmartinic @murtron what do you think?

Reply:
Agreed. We actually use it in a production system at the moment hence why we built it.

Move to a beta branch or something?